student.rb

```diff
@@ -1,5 +1,5 @@
 class Student < Person
-  attr_accessor :classroom
+  attr_accessor :classroom, :name
 
   def initialize(age:, name: 'Unknown', parent_permission: true, classroom: nil)
     super(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly add  a method that serves as the 'belongs-to' side (a student belongs to a classroom) just as the previous reviewer requested 👍🏿 

```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```